dev-packages/application-manager/src/generator/frontend-generator.ts

```diff
@@ -99,7 +99,7 @@ function start() {
     themeService.loadUserTheme();
 
     const application = container.get(FrontendApplication);
-    application.start();
+    return application.start();
```

Comment:
Left over, but I can extract it to its own commit because else the generated promise chain won't fully complete if an error occurs within `application.start`.

Reply:
that's fine with me in any way